GOVERNANCE.md

```diff
@@ -3,16 +3,31 @@
 This document describes the governance process under which the Serverless
 Working Group (WG) will manage this repository.
 
-## Working Group Meetings
+## Working Group Process
 
 In order to provide equitable rights to all Working Group members,
 the following process will be followed:
 
 * Official WG meetings will be announced at least a week in advance.
-* Proposed changes to any document will be done via a Pull Request (PR).
-* PRs will be reviewed during official WG meetings.
-* During meetings, priority will be given to PRs that appear to be ready for
-  a vote over those that appear to require discussions.
+* Proposed changes to any document will be done via a Pull Request (PR)
+* PRs will be reviewed in advance of WG meetings
+* Active members will vote on PRs in advance of Working Group meetings to
+  promote them to active consideration
```

Comment:
I don't know what "Active members" mean since we don't have levels of membership.

Reply:
Yes we do.  We have members with [voting rights](https://docs.google.com/spreadsheets/d/1bw5s9sC2ggYyAiGJHEk7xm-q2KG6jyrfBy69ifkdmt0/edit?pli=1#gid=0)